src/stepfunctions/steps/service.py

```diff
@@ -32,6 +33,14 @@ class DynamoDBApi(Enum):
     DeleteItem = "deleteItem"
     UpdateItem = "updateItem"
 
+class EKSApi(Enum):
```

Comment:
```suggestion
class EksApi(Enum):
```

nit: consistency

Reply:
Will include the change in the next commit